htmd/parameterization/cli.py

```diff
@@ -41,7 +41,8 @@ def getArgumentParser():
                         help='QM environment (default: %(default)s)')
     parser.add_argument('--no-min', action='store_false', dest='minimize',
                         help='Do not perform QM structure minimization')
-    parser.add_argument('--no-esp', action='store_false', dest='fit_charges', help='Do not perform QM charge fitting')
+    parser.add_argument('--charge-type', default='ESP', choices=['None', 'Gasteiger', 'ESP'],
```

Comment:
I like it. Maybe before the release we should add some deprecation warnings for old flags, but not important now.

Reply:
I have updated the docs. So it should be enough for now.